checker/tests/integration_tests.rs

```diff
@@ -149,6 +151,20 @@ fn invoke_driver(
     sys_root: String,
     extern_deps: Vec<(&str, String)>,
 ) -> usize {
+    // Configure callbacks. This reads the source to check for some configuration
```

Comment:
This is relevant to issue #52, please remember to add a comment to it when this is ready to merge.